packages/medusa/src/api/routes/store/swaps/create-swap.ts

```diff
@@ -26,6 +40,15 @@ import { defaultFields, defaultRelations } from "./"
  *                 quantity:
  *                   description: The quantity to return.
  *                   type: integer
+ *                 reason_id:
+ *                   description: The id of the reason of this return
+ *                   type: string
+ *                 note_id:
+ *                   description: The id of the note
+ *                   type: string
+ *               required:
+ *                 - item_id
+ *                 - quantity
```

Comment:
Maybe I missed something - does this work? I don't think we have used it in other places and would expect oas generation to fail validation because of this

Reply:
Correct, I don't think we use this somewhere in the codebase, however, I read about it [here](https://swagger.io/docs/specification/describing-request-body/). Please scroll to "Form Data" under the 3rd code snippet. I ran the build-openapi script and it seems to be working fine :+1: